src/transforms/babel.js

```diff
@@ -177,14 +192,40 @@ function mergeConfigs(a, b) {
   return a;
 }
 
-function hasPlugin(arr, plugin) {
-  return Array.isArray(arr) && arr.some(p => getPluginName(p) === plugin);
+function hasPlugin(arr, plugins) {
+  return (
+    Array.isArray(arr) && arr.some(p => plugins.includes(getPluginName(p)))
+  );
 }
 
 function getPluginName(p) {
   return Array.isArray(p) ? p[0] : p;
 }
 
+function testPluginArrayForBabelScope(plugins) {
+  for (let plugin of plugins) {
+    if (getPluginName(plugin).startsWith('@babel/')) {
```

Comment:
Does babel 7 require the scope or will they automatically add it if you just write `env`? If so, then this check won't work.

Reply:
Babel docs:

> You can still use the shorthand way of specifying a preset or plugin. However because of the switch to scoped packages, you still have to specify the @babel/ just like if you had your own preset to add to the config.

So yes it's required, although it's not 100% reliable as third party libraries will still have babel-....

Reply:
ok